src/System.Memory/ref/System.Memory.cs

```diff
@@ -117,5 +115,83 @@ public static class SpanExtensions
         
         public static ReadOnlySpan<TTo> NonPortableCast<TFrom, TTo>(this ReadOnlySpan<TFrom> source) where TFrom : struct where TTo : struct { throw null; }
     }
+
+    public struct ReadOnlyMemory<T>
```

Comment:
What about the System.Runtime ref?

Reply:
I amgoing to add that in a subsequent PR after this one is merged.

Reply:
Here is the PR: https://github.com/dotnet/corefx/pull/23841